### ISSUE TITLE
Move bonitoo tests to influxdb repo

### ISSUE THREAD

**Issue by desa** [influxdata/influxdb#18032]
The plan is to migrate the code from https://github.com/bonitoo-io/selenium-accept-infl2 into the influxdb repo.

**Comment by desa** [comment]
When this is implemented, be sure to communicate the plan to the bonitoo team.

**Comment by glinton** [comment]
Closed by #18090

**Comment by glinton** [comment]
For now, the process of keeping the repositories in sync can be as follows:

Whenever there is a pr merged to bonitoo-io, we should run the following in this repo:
```sh
curl https://patch-diff.githubusercontent.com/raw/bonitoo-io/selenium-accept-infl2/pull/<PR-NUMBER>.patch \
  | git apply --directory=e2e -
```

After a merge to master that affects code in `e2e` , update bonitoo's repo with a patch:
**get** (run in influxdb repo): `git diff <LAST_SYNCED_COMMIT> e2e`
**apply** (run in bonitoo repo): `git apply -p e2e <PATCH>`
